src/store/reducers/createPost.ts

```diff
@@ -0,0 +1,46 @@
+import * as actionTypes from 'store/actions/actionTypes';
+import { IPost } from 'types';
+
+
+const initialState = {
+    loading: false,
+    posts:[
+        {
+            title: 'First Post',
+            place: 'Rawalpindi',
+            description: 'This is my first post.',
+            owner: 'user1',
+        }
+    ]
+};
```

Comment:
place it in different files names initial state

Reply:
Placed in a separate file named initialValues.